x-pack/plugins/security_solution/public/management/pages/policy/view/policy_hooks.ts

```diff
@@ -101,3 +101,17 @@ export function usePolicyDetailsArtifactsNavigateCallback(listId: string) {
     [getPath, history]
   );
 }
+
+export const useIsPolicySettingsBarVisible = () => {
+  const policyTabs = [
```

Comment:
If you decide to keep this hook (see my prior comment about implementing a more generic solution), then consider using instead the Enums and const's we have for the paths below.

`administration/policy` ➡️  `POLICIES_PATH`

App level route consts here: https://github.com/elastic/kibana/blob/78d6acef2468d3f5e3187c21bab509f8b98b3885/x-pack/plugins/security_solution/common/constants.ts#L144

Management main page section Enum here (but I don't think you need this, since the above has a const for the full path): https://github.com/elastic/kibana/blob/ef3bd049793c97031cf3b0b7b0b4c935a3bec762/x-pack/plugins/security_solution/public/management/types.ts#L26-L36